### DISCUSSION TITLE
Why the   endpoints is not fully displayed in  endpoint page of skywalking v9.0.0

### DISCUSSION THREAD

**Discussion by czxin788** [apache/skywalking#8915] [Q&A] [answered]
I found that there  are some endpoints can be found in the trace page, but not foud in the endpoint page.
Can the endpoint page only display some  endpoints，but not all?

![image](https://user-images.githubusercontent.com/13861904/164169310-b9bb99f5-59d0-48fe-9f58-95f0c029d283.png)

**Comment by wu-sheng** [comment, marked as answer]
Run the search, endpoint is never fully listed, it could be 100k+ per service. Search them.
